### PR TITLE
Adjusting for new vogue config

### PR DESCRIPTION
**Pull request by mayabrandi**
This PR addjusts cg to run vogue with config. 

**How to prepare for test**:
On hasta

1.  cd /home/proj/stage/servers/resources/hasta.scilifelab.se

2. bash update-servers-stage.sh vogue_alias

3. bash update-cg-stage.sh vogue_config

4. bash update-vogue-stage.sh master

5. run:

```
usestage
cg upload vogue genotype -d 300
```

**Expected test outcome**:
Something like this:
```
----------------- UPLOAD ----------------------
----------------- TRENDING -----------------------
----------------- GENOTYPE -----------------------
vogue output: INFO:vogue.adapter.plugin:Use database vogue-stage.
vogue output: INFO:vogue.load.genotype:Trying to build json
vogue output: INFO:vogue.adapter.plugin:No updates for sample ADM1464A1.
...
```

**Review:**
- [X] code approved by @barrystokman 
- [x] tests executed by MB
- [x] "Merge and deploy" approved by @barrystokman 
Thanks for filling in who performed the code review and the test!

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by mayabrandi** [comment]
Testing acording to above and the outcome was as expected. Aborted cg load genotype after 7 documents were added. The 7 documents could then be found in the new mongo database.

Skärmavbild 2020-05-12 kl  13 32 16

Skärmavbild 2020-05-12 kl  13 32 28

**Comment by mayabrandi** [comment]
@patrikgrenfeldt ready for review and deploy? Ill run black after the code has been reviewed

File: tests/apps/vogue/conftest.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 import pytest
 from cg.apps.vogue import VogueAPI
 
-CONFIG = {"vogue": {"binary_path": "gtdb"}}
+CONFIG = {"vogue": {"binary_path": "gtdb", "config_path": "vogue_config"}}
```
Comment (henrikstranneheim) [inline tests/apps/vogue/conftest.py:8]: What binary is "gtdb"?

**Comment by mayabrandi** [comment]
It is nothing. Just a dummy. I changed it to "/path/to/vogue”

**Comment by hassanfa** [comment]
Note: PR #627 is depending on this PR to be merged.

**Comment by barrystokman** [comment]
Bumped:
image

Deployed:
image

**Pull request merged**